packages/store/src/patterns/patternMatchers.js

```diff
@@ -36,10 +37,80 @@ import {
 
 const { quote: q, details: X } = assert;
 const { entries, values } = Object;
+const { ownKeys } = Reflect;
 
 /** @type WeakSet<Pattern> */
 const patternMemo = new WeakSet();
 
+// /////////////////////// Match Helpers Helpers /////////////////////////////
+
+/**
+ * exported primarily for testing.
+ */
+export const defaultLimits = harden({
```

Comment:
That's what I intend `decimalDigitsLimit` to be. Should the name be changed to avoid a misunderstanding? This is the only limit name doing double duty. It applies both to bigints by themselves, and it applies to the cardinality counts of copyBags.

Reply:
I'm resolving this. But if you think I do need to change the name, please unresolve.